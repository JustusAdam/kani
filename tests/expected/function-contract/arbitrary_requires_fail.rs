// Copyright Kani Contributors
// SPDX-License-Identifier: Apache-2.0 OR MIT
// kani-verify-fail

fn div(dividend: u32, divisor: u32) -> u32 {
    dividend / divisor
}

#[kani::proof]
fn main() {
    div(kani::any(), kani::any());
}
