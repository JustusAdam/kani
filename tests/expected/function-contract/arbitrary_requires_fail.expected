RESULTS:
Check 1: div.assertion.1
	 - Status: FAILURE
	 - Description: "attempt to divide by zero"

Check 3: div.division-by-zero.1
	 - Status: SUCCESS
	 - Description: "division by zero"
